tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -380,6 +397,10 @@ public boolean init(String[] args) throws ParseException, IOException {
     // src_dir & hdfs_classpath flags are for compatibility.
     srcDir = cliParser.getOptionValue("src_dir");
 
+    applicationTags = convertStringToValueSet(
```

Comment:
Rather than adding a `convertStringToValueSet`, you could also just use
```
new HashSet(tonyConf.getStringCollection(...);
```

Reply:
Addressed